packages/email/email_tests.js

```diff
@@ -37,7 +49,57 @@ Tinytest.add("email - dev mode smoke test", function (test) {
                "of the message\r\n" +
                "From us.\r\n" +
                "====== END MAIL #0 ======\n");
+  });
+});
 
+Tinytest.add("email - undefined headers sends properly", function (test) {
+  smokeEmailTest(function (stream) {
+    Email.send({
+      from: "foo@example.com",
+      to: "bar@example.com",
+      subject: "This is the subject",
+      text: "This is the body\nof the message\nFrom us.",
+    });
+
+    test.matches(stream.getContentsAsString("utf8"),
+      /^====== BEGIN MAIL #0 ======$[\s\S]+^To: bar@example.com$/m);
+  });
+});
+
+Tinytest.add("email - multiple e-mails same stream", function (test) {
+  smokeEmailTest(function (stream) {
+    // Test if date header is automaticall generated, if not specified
```

Comment:
This comment does not look correct. The date header is not checked here and the tests checking the auto generated date header is further down and has the same comment.